### PR TITLE
Reinstated cacheBuster to new-client

### LINKED ISSUE #1323: hfetch - reinstate the cacheBuster functionality.
The cacheBuster fuctionality was disabled little over a year ago because we updated to webpack 5 which does not have support for process.env.

The cacheBuster fuctionality in hfetch should be reinstated again.... somehow.... with or without process.env.

Today, the code is just commented, and forgotten.

`this.useCacheBuster = process?.env?.REACT_APP_USE_CACHE_BUSTER === "true" || false;`

### PR DESCRIPTION
**Pull request by jesade-vbg**
Closes #1323

### CODE REVIEW

File: new-client/src/utils/FetchWrapper.js [hunk 1]
```diff
@@ -57,6 +59,14 @@ class FetchWrapper {
...
   }
 
+  getMetaValue(key) {
+    const el = document.getElementsByName(key);
+    if (el && el[0]) {
+      return el[0]?.attributes?.content?.value;
+    }
+    return null;
+  }
+
```
Comment (jacobwod) [inline new-client/src/utils/FetchWrapper.js:69]: Is this really necessary, wouldn't something like this be more sufficient and work on all modern browsers anyway?
```javascript
document.querySelector("meta[name='hajk-client-use-cache-buster']").getAttribute("content");
```

File: new-client/src/utils/FetchWrapper.js [hunk 2]
```diff
@@ -39,10 +39,12 @@ class FetchWrapper {
     this.options = {};
 
     // Hash is used for cacheBuster function.
-    //this.hash = process?.env?.REACT_APP_GIT_HASH || null;
-    this.hash = null;
-    //this.useCacheBuster = process?.env?.REACT_APP_USE_CACHE_BUSTER === "true" || false;
-    this.useCacheBuster = false;
+    // Lets get the values from generated meta-tags
+    this.hash = this.getMetaValue("hajk-client-git-hash") || "";
```
Comment (jacobwod) [inline new-client/src/utils/FetchWrapper.js:43]: Can we be sure that the meta tag will always be called `hajk-client-use-cache-buster`? I mean, it's created during build time by parsing `%REACT_APP_NAME%-use-cache-buster`, so another value on `REACT_APP_NAME` will break this completely.

**Comment by jesade-vbg** [comment]
You're absolutely right regarding these 😂 I should never make PR:s on Friday afternoons 😂  I'll fix on Monday.

**Comment by jesade-vbg** [comment]
Now it's corrected. Thanx

**Review by jacobwod — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**